[2022-02-28 00:00:03,413] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-02-27T18:00:00+00:00 [queued]>
[2022-02-28 00:00:03,420] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-02-27T18:00:00+00:00 [queued]>
[2022-02-28 00:00:03,420] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 00:00:03,420] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-02-28 00:00:03,420] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 00:00:03,431] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): enter_virtual_env> on 2022-02-27T18:00:00+00:00
[2022-02-28 00:00:03,435] {standard_task_runner.py:52} INFO - Started process 134410 to run task
[2022-02-28 00:00:03,442] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'matchup', 'enter_virtual_env', '2022-02-27T18:00:00+00:00', '--job-id', '97', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/matchup.py', '--cfg-path', '/tmp/tmp5zl4hgml', '--error-file', '/tmp/tmpgq15p3ln']
[2022-02-28 00:00:03,443] {standard_task_runner.py:77} INFO - Job 97: Subtask enter_virtual_env
[2022-02-28 00:00:03,512] {logging_mixin.py:104} INFO - Running <TaskInstance: matchup.enter_virtual_env 2022-02-27T18:00:00+00:00 [running]> on host 531edfb863fc
[2022-02-28 00:00:03,565] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=r.chesne@gmail.com
AIRFLOW_CTX_DAG_OWNER=Robin
AIRFLOW_CTX_DAG_ID=matchup
AIRFLOW_CTX_TASK_ID=enter_virtual_env
AIRFLOW_CTX_EXECUTION_DATE=2022-02-27T18:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-27T18:00:00+00:00
[2022-02-28 00:00:03,566] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-02-28 00:00:03,567] {bash.py:158} INFO - Running command: . /app/env/bin/activate
[2022-02-28 00:00:03,578] {bash.py:169} INFO - Output:
[2022-02-28 00:00:03,587] {bash.py:177} INFO - Command exited with return code 0
[2022-02-28 00:00:03,641] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=matchup, task_id=enter_virtual_env, execution_date=20220227T180000, start_date=20220228T000003, end_date=20220228T000003
[2022-02-28 00:00:03,668] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-28 00:00:03,691] {local_task_job.py:146} INFO - Task exited with return code 0
